src/utils/theme.js

```diff
@@ -41,6 +41,18 @@ export const zIndexes = {
   gridDebugger: 10001
 }
 
+const colorsVariables = {
```

Comment:
here I just renamed the variables according to our zeplin doc (https://app.zeplin.io/project/5b4e666f0d4230a16dffd93a/screen/5c61564dd958777e20e8faec), however some more colours were added so I named them the best that I could :/